A2-service-configs-in-dns.md

```diff
@@ -101,16 +104,17 @@ When encoding the service config in DNS, TXT records are the "obvious"
 choice, since the service config is effectively additional metadata
 associated with the DNS name.
 
-## Implementation
+We use the `_grpc_config` prefix for the DNS entry to allow a service
+config to be specified for a service whose primary record is a CNAME
+record, because DNS does not allow any other record for a same name
+that contains a CNAME record.
 
-The implementation will be done in C-core first.  Once the new c-ares
-DNS resolver code (https://github.com/grpc/grpc/pull/7771) has been
-merged, we will extend it to query for the TXT records and return the
-resulting service config JSON data.
+## Implementation
 
-Note that, due to platform support issues, we will initially *not*
-support the c-ares resolver under Windows or for Node.  Alternatives
-will need to be found for these environments.
+The implementation has been done in C-core as part of the c-ares DNS
+resolver.  We are currently working toward making the c-ares resolver
+the default DNS resolver for C-core.  This requires things like Windows
+and node support and adding address sorting.
```

Comment:
s/node/Node
And why node is special here? Does it use c-core?

Reply:
Done.

We are in the process of working on a native javascript implementation for Node, but the comment here refers to Node support in C-core.  There are some issues related to using c-ares with Node that we still need to work on.